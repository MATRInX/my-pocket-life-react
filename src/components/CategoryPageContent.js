import React from 'react';

const CategoryPageContent = (props) => (
  <div>
    CategoryPageContent - category:
    <p>{props.match.params.category}</p>
  </div>
);

export default CategoryPageContent;