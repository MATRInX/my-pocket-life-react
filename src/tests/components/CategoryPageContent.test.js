import React from 'react';
import { shallow } from 'enzyme';
import CategoryPageContent from '../../components/CategoryPageContent';

let wrapper, category, match;

beforeEach(() => {
  category = 'DIY';
  match = {
    params: {
      category
    }
  };
  wrapper = shallow(<CategoryPageContent match={match}/>);
})

test('should correctly render CategoryPageContent', () => {  
  expect(wrapper).toMatchSnapshot();
});

test('should correctly show selected category', () => {
  const categoryNode = wrapper.find('p');
  expect(categoryNode.text()).toEqual(category);
});