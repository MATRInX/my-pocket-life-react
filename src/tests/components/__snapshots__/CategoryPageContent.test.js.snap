// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render CategoryPageContent 1`] = `
<div>
  CategoryPageContent - category:
  <p>
    DIY
  </p>
</div>
`;
